scribejava-apis/src/main/java/com/github/scribejava/apis/FlickrApi.java

```diff
@@ -10,7 +10,16 @@
  */
 public class FlickrApi extends DefaultApi10a {
 
-    protected FlickrApi() {
+    private static final String AUTHORIZE_URL = "https://www.flickr.com/services/oauth/authorize?oauth_token=%s";
+
+    private String perm; /* read, write, or delete (delete includes read/write) */
```

Comment:
1.maybe enum?
Is it closed set of value? do you have a link to the docs?
2.maybe final. So FlickrApi will stay immutable
3.is it required param? Maybe a batch of methods will be better?
public static FlickrApi read() {
public static FlickrApi write() {
public static FlickrApi delete() {
And user will choose which one to use.